src/network/models/configuration.rs

```diff
@@ -0,0 +1,179 @@
+// (C) Copyright IBM Corp. 2024.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fmt::Display;
+
+use crate::{errors::DeserializationError, Error, Result};
+use serde::Deserialize;
+
```

Comment:
Diff is not friendly here, but I just copied everything from `models.rs` into this `configuration.rs` and `metering.rs` files. There are no other changes.